Packs/FeedCyrenThreatInDepth/TestPlaybooks/CyrenThreatInDepth-Test.yml

```
@@ -0,0 +1,141 @@
+id: CyrenThreatInDepth-Test
+version: 4
```

Internal version is always -1
```suggestion
version: -1
```